tests/JobCreatorTest.php

```diff
@@ -1949,4 +1949,46 @@ public function provideComposerInstall(): array
             ],
         ];
     }
+
+    public function testDuplicateJobsRemoved(): void
+    {
+        if (!function_exists('yaml_parse')) {
+            $this->markTestSkipped('yaml extension is not installed');
+        }
```

Comment:
We really shouldn't skip these - it should just be a requirement that yaml_parse is installed if you wanna run tests. OTherwise it looks like everything's fine when stuff is just being skipped but might be broken.

There's more than just this one test doing it though so that's a discussion to be held outside this PR.